readme.md

```diff
@@ -55,7 +55,8 @@ Configure it in `package.json`.
 			"unicorn/error-message": "error",
 			"unicorn/no-unsafe-regex": "off",
 			"unicorn/prefer-add-event-listener": "error",
-			"unicorn/no-console-spaces": "error"
+			"unicorn/no-console-spaces": "error",
+			"no-unreadable-array-destructuring": "error"
```

Comment:
Isn't this line missing a "unicorn/" prefix?

Reply:
Yup, thanks for noticing: https://github.com/sindresorhus/eslint-plugin-unicorn/commit/6bd6156ba1ddcd0675e7a8fee75f4bbe2b819f92